models/pull.go

```diff
@@ -60,15 +60,15 @@ type PullRequest struct {
 	Issue   *Issue `xorm:"-"`
 	Index   int64
 
-	HeadRepoID   int64       `xorm:"INDEX"`
-	HeadRepo     *Repository `xorm:"-"`
-	BaseRepoID   int64       `xorm:"INDEX"`
-	BaseRepo     *Repository `xorm:"-"`
-	HeadUserName string
-	HeadBranch   string
-	BaseBranch   string
-	MergeBase    string `xorm:"VARCHAR(40)"`
-
+	HeadRepoID     int64       `xorm:"INDEX"`
+	HeadRepo       *Repository `xorm:"-"`
+	BaseRepoID     int64       `xorm:"INDEX"`
+	BaseRepo       *Repository `xorm:"-"`
+	HeadUserName   string
+	HeadBranch     string
+	BaseBranch     string
+	MergeBase      string         `xorm:"VARCHAR(40)"`
+	LastCommitID   string         `xorm:"-"`
```

Comment:
Not sure about the styleguide here, but shouldn't the tags be aligned, and why 3 spaces between `LastCommitID` and `string`?